turbopy/core.py

```diff
@@ -411,33 +425,38 @@ def __init__(self, owner: Simulation, input_data: dict):
         self._module_type = None
         self._input_data = input_data
 
-    def publish_resource(self, resource: dict):
```

Comment:
I think the build is failing because this method was removed. I would like to see if we can keep this method, so that any modules written with the "old" sharing API will still work.